include/mysql.h

```diff
@@ -189,7 +189,10 @@ enum mysql_option
   /* MariaDB options */
   MYSQL_PROGRESS_CALLBACK=5999,
   MYSQL_OPT_NONBLOCK,
-  MYSQL_OPT_USE_THREAD_SPECIFIC_MEMORY
+  MYSQL_OPT_USE_THREAD_SPECIFIC_MEMORY,
+
+  /* Libmariadb compatibility */
+  MARIADB_OPT_TLS_VERSION
```

Comment:
Backup utility exposes the tls version option though the cmd but didn't setup it during the connection. I suggest to use it for compatibility. If no, hiding it for the backup util is also an option.